### DISCUSSION TITLE
How do I move a file to a desired folder in lazyvim.

### DISCUSSION THREAD

**Discussion by ghost** [LazyVim/LazyVim#2489] [Q&A]
Using a neotree setup to manage files. I am new to using lazyvim. Please direct me to help. THANKS !

**Comment by cerebrl** [comment]
I would direct this question to the [Neotree project](https://github.com/nvim-neo-tree/neo-tree.nvim). This isn't really a LazyVim question. If you didn't know, you can press the `?` key while in Neotree, and you should get a help menu pop up that helps discover the available actions.

**Comment by ghost** [comment]
@cerebrl that doesn't answer the issue. What I mean is how does one move a file itself to the folder. Not the other way around.

**Comment by LoftyShadow** [comment]
![image](https://github.com/LazyVim/LazyVim/assets/43536131/4aad8108-35b8-4592-83a1-c9f7dc02d737)
![image](https://github.com/LazyVim/LazyVim/assets/43536131/6e5569d6-8329-4776-90b9-9222e975acf3)
